### DISCUSSION TITLE
embedding a draw.io diagram into confluence and show a specific detail of the whole diagram.

### DISCUSSION THREAD

**Discussion by Harald0711** [jgraph/drawio#4500] [Ideas]
Since I often have quite large diagrams, I have the use case that I want to display specific parts of this diagram on different Confluence pages. In other words, the diagram should be displayed, but only an enlarged section of the whole diagram. 
I know that you can set up which layers should be displayed in the viewer. 
Unfortunately, this does not help me for my use case. Even in this case, the viewer only shows certain layers, but always the entire diagram. 
As a user, I can use the viewer to zoom and move the image so that the desired part is displayed. It would be ideal if this setting could be saved for the embedded diagram. When the Confluence page is called up again later, the zoomed section of the diagram should be displayed directly. 
If this option were provided in the draw.io viewer macro for Confluence, that would be great.

**Comment by AJBuilder** [comment]
I have this use-case all the time.
Its really annoying having to screenshot certain sections and embed the image. When you change the diagram, you have to change all the screenshots manually.

**Comment by davidjgraph** [comment]
For confluence issue please submit this as a standard support ticket.

**Reply by Harald0711** [reply]
I'm a bit confused. I already done that. Link to the [community post](https://community.atlassian.com/t5/App-Central-questions/How-can-I-embed-a-draw-io-diagram-and-show-a-specific-detail-of/qaq-p/2731950?utm_source=atlcomm&utm_medium=email&utm_campaign=mentions_reply&utm_content=topic#U2735698). 
Someone from the Confluence community sent me here and wrote that I should submit the idea for further development to the development team. Thats what I did. 
For me, this is an additional requirement that is not available in the draw.io viewer macro for confluence today. 
If the development team could take up the idea (however), then I would be very happy.

**Reply by davidjgraph** [reply]
Ah, I follow you. But if you're a paying customer submit a ticket at https://marketplace.atlassian.com/apps/1210933/draw-io-diagrams-whiteboards?hosting=cloud&tab=support so we know this is a request by a paying user.

**Reply by Harald0711** [reply]
OK, then I'll follow your advice. As my company is a paying customer, I will also create a support ticket there. I hope it helps that the request can be implemented ;-)  
Many thanks for the advice.

**Reply by davidjgraph** [reply]
Thanks for your email, your license has been confirmed. We'll move your request into issues and label as a feature request. Internal ref FV9-7TW
